docs/src/getting_started.md

```diff
@@ -1,17 +1,17 @@
 # Getting Started
```

Comment:
For a later PR: I think this page needs an overhaul. We should show how to install julia and get climaland as described, but then show how easy it is to run a simulation (using the simulations struct + plot(simulation), and all default parameterizations) in a few lines of code. 

I think all of the rest can go into a different file or page